config/locales/pt.yml

```diff
@@ -191,13 +191,33 @@ pt:
   create_new_community: "Criar uma Nova Comunidade"
 
   # Mail messages
-  hello: "Olá %{user_name}"
-  new_user: "Obrigado pelo seu contacto. A sua mensagem foi recebida pela nossa equipa de suporte e iremos entrar em contacto consigo o mais rapidamente possível."
-  account_generated: "Foi criada uma nova conta para si. Para fazer login, use este link:"
-  above_this_line: "Escreva a sua resposta acima desta linha"
-  message_id: "Id da Mensagem"
-  message_response: "A seguinte mensagem foi criada em resposta à sua questão:"
-  view_online: "Ver isto online:"
```

Comment:
@popperz0r Tiago, Sorry again for the delay.  I am curious if you really meant to replace the PT translations with English (these are the strings you sent me via Twitter).